koku/masu/processor/aws/aws_report_db_cleaner.py

```diff
@@ -106,8 +105,7 @@ def purge_expired_report_data(self, expired_date=None, provider_uuid=None, simul
 
         """
         LOG.info("Calling purge_expired_report_data for aws")
-        with ReportingCommonDBAccessor() as reporting_common:
-            column_map = reporting_common.column_map
+        column_map = REPORT_COLUMN_MAP
```

Comment:
Why does this code take column_map in it's constructor?

```with AWSReportDBAccessor(self._schema, column_map) as accessor:
            if (expired_date is None and provider_uuid is None) or (  # noqa: W504	            if (expired_date is None and provider_uuid is None) 
```

It's a constant so I think we can remove column maps from all constructors.

Reply:
updated